.env

```diff
@@ -0,0 +1,2 @@
+SCICAT_API_BASE_PATH=
+SCICAT_API_ACCESS_TOKEN=
```

Comment:
Please do not expose `.env` file, you are risking the token to be uploaded by accident. `.env` should be included in the `.gitignore`
For example demonstration you can use `.env.example`